test_scripts/Defects/4_5/1921_Invalid_PT_after_cutting_unknow_values.lua

```diff
@@ -0,0 +1,126 @@
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = {"gps"}
+      },
+      SubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        newParameter = {"value"}
+      },
+      UnknownAPI = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = {"gps"}
+      },
+      SendLocation = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"}
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups =
+  {"Base-4", "NewTestCaseGroup"}
+end
+
+local function NotValidPtuUpdateFunc(tbl)
+  local VDgroup = {
+    rpcs = {
+      GetVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = {"gps"}
+      },
+      SubscribeVehicleData = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        newParameter = {"value"}
+      },
+      UnknownAPI = {
+        hmi_levels = {"BACKGROUND", "FULL", "LIMITED"},
+        parameters = {"gps"}
+      },
+      SendLocation = {
+        -- missed mandatory hmi_levels parameter
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings["NewTestCaseGroup"] = VDgroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups =
+  {"Base-4", "NewTestCaseGroup"}
+end
+
+local function CheckCuttingUnknowValues(self)
+  commonDefects.rai_ptu_n_without_OnPermissionsChange(1, ptuUpdateFunc,self)
+  self.mobileSession1:ExpectNotification("OnPermissionsChange")
+  :Times(2)
+  :ValidIf(function(_,data)
+      local ErrorStatus = false
+      local ErrorMessage = ""
+      if #data.payload.permissionItem ~= 0 then
+        for i=1, #data.payload.permissionItem do
+          if data.payload.permissionItem[i].rpcName == "UnknownAPI" then
+            ErrorStatus = true
+            CuttingUnknownStatus = true
+            ErrorMessage = ErrorMessage .. " OnPermissionsChange contains unknown_RPC value.\n"
+          end
+          if data.payload.permissionItem[i].newParameter then
+            ErrorStatus = true
+            CuttingUnknownStatus = true
+            ErrorMessage = ErrorMessage .. " OnPermissionsChange contains unknown_parameter value.\n"
+          end
+        end
+      else
+        ErrorStatus = true
+        ErrorMessage = ErrorMessage .. "OnPermissionsChange is not contain permissionItem elements"
+      end
+      if ErrorStatus == true then
+        return false, ErrorMessage
+      else
+        return true
+      end
+    end)
+  commonFunctions:userPrint(33, "Check PTU content of PTU attempt in SDL log by message 'PTU content is'.\n"
+    .. "'policy_table' must not contain 'newParameter' and 'UnknownAPI' values.")
+end
+
+local function InvalidPTAfterCutingUnknownValues(self)
+  if CuttingUnknownStatus == true then
```

Comment:
@GetmanetsIrina Code 
```
if CuttingUnknownStatus == true then
self:FailTestCase(<.....>)
...
```
can make some confusion because by the name of the variable `CuttingUnknownStatus`, it may seem that unknown parameters have been cut.  In other words - everything is ok, but execution is failing script.
I would propose to change name of variable to more descriptive one, for example `isCuttingUnknownError`.